src/command_update.rs

```diff
@@ -75,6 +83,9 @@ fn update_channel(
                 );
             }
         }
+        JuliaupConfigChannel::AliasChannel { .. } => {
+            bail!("Internal error: Tried to update an alias channel '{channel}' directly.");
+        }
```

Comment:
Not sure I understand this error. Why can't we update through the alias? Maybe I don't follow but I thought there was even a test for this